src/factor.h

```diff
@@ -50,7 +52,7 @@ class CompactFactor {
    */
   CompactFactor(factor_id_t id);
 
-#define POTENTIAL(func_id) _potential_##func_id
+#define POTENTIAL_SIGN(func_id) _potential_sign_##func_id
```

Comment:
Love this rename!  There are multiple `potential()` in `CFG` and `CF` but I guess they are consistent.